InvenTree/InvenTree/metadata.py

```diff
@@ -93,11 +93,11 @@ def determine_metadata(self, request, view):
 
             # Add a 'DELETE' action if we are allowed to delete
             if 'DELETE' in view.allowed_methods and check(user, table, 'delete'):
-                actions['DELETE'] = True
+                actions['DELETE'] = {}
 
             # Add a 'VIEW' action if we are allowed to view
             if 'GET' in view.allowed_methods and check(user, table, 'view'):
-                actions['GET'] = True
+                actions['GET'] = {}
```

Comment:
comment: this is pretty deep in our meta logic.
@SchrodingersGat could you please take a look?

Reply:
@martonmiklos what is the rationale for this particular change?

Reply:
> @martonmiklos what is the rationale for this particular change?

Well someone recommended me to do that ;)
https://github.com/inventree/InvenTree/issues/3384#issuecomment-1198868669

Reply:
@martonmiklos I thought it sounded familiar. Thanks for the reminder :)

Reply:
> @martonmiklos I thought it sounded familiar. Thanks for the reminder :)

Getting back to the business: I am not familiar with the django to say it is safe or not, please decide if it can be handled differently or it is acceptable.

Reply:
The changes here as suggested by me should be acceptable